src/Auth.ts

```diff
@@ -83,6 +85,7 @@ export enum CertificateType {
 }
 
 export class Auth {
+  private _open: any;
```

Comment:
Instead of defining `_open` as `any`, we can add `import type * as open from 'open';` on top to load the `open` typings and then define `_open` as `typeof open`. That way, we'll get design-time type safety while lazy loading it.